### DISCUSSION TITLE
FastifyLoggerOptions type incomplete?

### DISCUSSION THREAD

**Discussion by enricoschaaf** [fastify/fastify#3313] [Q&A]
Hi 👋🏽

When trying to customize the logger with fastify there is according to the types not the option to configure the base, But according to pino this is an option. https://github.com/pinojs/pino/blob/master/docs/api.md#base-object
Is this option missing from the type definitons by accident or is it a deliberate choice?

Happy to provide a PR if it's just missing. 😊

**Comment by climba03003** [comment]
It is by design not to provide the whole types of `pino` in `fastify`.

Since `pino@6` do not ship with official types, as a TypeScript users I recommend you to install `pino@6` and `@types/pino` directly and pass the instance to `fastify`.
